src/Microsoft.PowerShell.Commands.Management/cimSupport/cmdletization/cim/cimConverter.cs

```diff
@@ -425,7 +425,9 @@ internal static object ConvertFromCimToDotNet(object cimObject, Type expectedDot
                 var cimIntrinsicValue = (byte[])LanguagePrimitives.ConvertTo(cimObject, typeof(byte[]), CultureInfo.InvariantCulture);
                 return exceptionSafeReturn(delegate
                                                {
+                                                   #pragma warning disable SYSLIB0057
```

Comment:
file issue to move in vNext

Reply:
filed: https://github.com/PowerShell/PowerShell/issues/24280